tests/atoms/gaia_apps.js

```diff
@@ -372,12 +372,31 @@ var GaiaApps = {
     };
     return result;
   },
+  /**
+  * Install the app with the specified ManifestURL
+  */
+  installWithManifestURL: function(manifestURL) {
```

Comment:
this change actually gave a failure on functionThatCreateRequest:
TEST-UNEXPECTED-ERROR | test_homescreen_launch_app.py TestLaunchApp.test_launch_app | JavascriptException: JavascriptException: TypeError: functionThatCreateRequest is not a function
stacktrace:
	execute_async_script @gaia_test.py, line 156
	inline javascript, line 395
	src: "    let req = functionThatCreateRequest(functionArgument);"